_overviews/scala3-book/scala-tools.md

```diff
@@ -301,7 +301,7 @@ version := "0.1"
 scalaVersion := "{{site.scala-3-version}}"
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "3.3.0-SNAP3" % Test
+  "org.scalatest" %% "scalatest" % "{{site.scala-3-version}}" % Test
```

Comment:
I don't think that's correct, scalatest doesn't follow the same version numbering as scala. I think it is good to change it to a non-SNAP version. I think version `3.2.6`, currently the latest version, works with Scala 3.